### ISSUE TITLE
Crash at startup

### ISSUE THREAD

**Issue by ASchneiderBR** [royshil/obs-backgroundremoval#50]
Hello!

Latest build now crashes my OBS at startup, unfortunately.

[Crash 2021-11-04 07-55-09.txt](https://github.com/royshil/obs-backgroundremoval/files/7474232/Crash.2021-11-04.07-55-09.txt)

No problem, removing the plugin solves the problem.

Thanks!

**Comment by royshil** [comment]
ok let me check. it's probably the version of libOBS ...
i'm rebuilding the windows release

**Comment by royshil** [comment]
@ASchneiderBR  please try the latest build: https://github.com/royshil/obs-backgroundremoval/releases/download/v0.4.0/obs-backgroundremoval-0.4.0-win64.zip

**Comment by ASchneiderBR** [comment]
Hello! I was in fact using this version. The problem was that I had this filter already applied to a webcam source, with the previous built. When I unpack the latest, OBS crashes at start up. Then I've removed the plugin, removed the filter from the webcam source, closed OBS, extracted the latest version and now it works! It seems that the filter already applied with a previous version, and enabled, did not allowed OBS to start, not sure why. I'm sure I've downloaded the latest build zip file and overwritten everything, really odd. Thanks!
